src/gui/application.cpp

```diff
@@ -724,6 +740,32 @@ void Application::setHelp()
     _helpOnly = true;
 }
 
+void Application::parseEditLocalFileArgument()
```

Comment:
this name is confusing as we are not parsing anything (already done inside `parseOptions` maybe use `handleEditLocallyFromOptions` ? or something shorter but with `handle` in the name